### PR TITLE
Vishal/mainnet16nodes

### PR DESCRIPTION
**Pull request by vishalchangrani**
Closes: #???

## Description

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: nodeoperators/NodeOperatorList.md [hunk 1]
```diff
@@ -56,7 +56,6 @@
 | Bitcoin Louie | Verification | fffba108a52e4ca8f5d30658c3f03f47ad1cc13a0995435c0532336236813218
 | Blockchain at Berkeley | Verification | 8f8d77ba98d1606b19fce8f6d35908bfc29ea171c02879162f6755c05e0ca1ee
 | Blockchain Coalition/Block Venture | Verification | 16036263d71f64d65f84542e9f7678e578482a107b61e45b88759ebea2c7451d
-| Blockchange | Verification | e6b03a5cea7568966d4326d9c4191695b450aa5a3095f4ddbd6957643b23538e
```
Comment (vishalchangrani) [inline nodeoperators/NodeOperatorList.md:59]: blockchange have unstaked their VN

File: nodeoperators/NodeOperatorList.md [hunk 2]
```diff
@@ -118,7 +117,6 @@
 | Metapurse | Consensus |1780288437c9aed5056836bfffcfe19334dd6e229607fccfb3fe9b1d4d24cdca
 | Metapurse | Verfication | dc3054cae816874d7dbbe838a41ef97c8a01b44643bffc82afe3a8228eed29fd
 | NFT42 | Verification | f8af31c7c3294b246f34040e5685c351e89f1c1edf6c64f93f7babd73d149951
-| Optakt | Consensus | f3b72c50794c29319f0f77f0c7d623662eb15253c291a5360d095a4ea282fadb
```
Comment (vishalchangrani) [inline nodeoperators/NodeOperatorList.md:121]: Max W will not be running a node this time

File: nodeoperators/nodelist.json [hunk 3]
```diff
@@ -3638,5 +3622,61 @@
         "Stake": 1000,
         "NetworkPubKey": "52bfa35f885c0d56068596b8823b827dab5a2414682b62caaa709de0e6a20c721299316a6cf1aa54035fa79b02c840f402ce672aaff4080b41db175c44b0ea64",
         "StakingPubKey": "9900bd5c3160660918fc228fe2e19769620f115fa7f6a9b0ba1c8bc229cb80304868c69b71c639bdae271fff96ab1baa13d4194b6dad5ee95a46cd96c7c417a3ff9a1cf8b0baa6fc0653b5c68c6e4428adcbc6d75579306d02897a161027a9c2"
+    },
+    {
+        "Role": "access",
```
Comment (vishalchangrani) [inline nodeoperators/nodelist.json:3627]: Bitture 1 AN

File: nodeoperators/nodelist.json [hunk 4]
```diff
@@ -3638,5 +3622,61 @@
...
+        "Address": "flow-access-node.bitrue.com:3569",
+        "NodeID": "ac7219d3c60bd98cb8005b80439751fb707ab4305ac9298c71dc957a04a8f58b",
+        "Stake": 1000,
+        "NetworkPubKey": "ecc07fd9fbec9d0a8ac2db61c65ce7d87ebafc8132bf7ba7e33a1d0c44948e19df8dcffa4735b37236939918e7811c2f0a69e2f8f0463cc445dbf6f2314308ea",
+        "StakingPubKey": "96fa2f132dc7e1e2f9110a38890c5cf6d245a16360ef4016bb5c7513890c9efface098ebb2af7ef8f358beed27eea07f0ecce6e6eac18629f018ccf92f3fcb27ddaf2bd7b19ca282314b0bd0cec099f6ea2059c86208719060cf0aa1f0aa0e49"
+    },
+    {
+        "Role": "access",
```
Comment (vishalchangrani) [inline nodeoperators/nodelist.json:3635]: Ginco 3 ANs

File: nodeoperators/nodelist.json [hunk 5]
```diff
@@ -3638,5 +3622,61 @@
...
+        "Address": "prod173.auroraplatform.com:3569",
+        "NodeID": "1f0c51ffb1c6e82b2f42982678c923ca029152a6bdc6a217fe2b7ae01de105a5",
+        "Stake": 1000,
+        "NetworkPubKey": "fda3b9c2ae65f1721e3b79ac015e7b66c7276048e6d9bd5cfd92801cc5ad3b10283ecba70e67e9771bbb9d723a01799f6386576793447126002d111daa747235",
+        "StakingPubKey": "96b348c154387006d0400441f0588eea19587c0978ef89f45fb325aaf153d7c5cde6d5024af6a5bf9c041fa90a403798087f65f208a239ecddd6fcaacf8f71b7cc33aff5a7f16f8e9ed73c2c646ffe8f4dd5b8e0cdb06f7ecd3694325354c876"
+    },
+    {
+        "Role": "access",
```
Comment (vishalchangrani) [inline nodeoperators/nodelist.json:3667]: Bison 1 AN

File: nodeoperators/nodelist.json [hunk 6]
```diff
@@ -3638,5 +3622,61 @@
...
+        "Address": "p2p.f5a22b04-1276-447a-ac81-4a626129d590.flow.bison.run:3569",
+        "NodeID": "f58a1e6416fad4267bcaf7993d1300f85ba618aa3b7e69a7009d21a2a94c3114",
+        "Stake": 1000,
+        "NetworkPubKey": "0fa75eb2b0109badcf0aca87add84d48a659c05081efbd780fa94039580f633a1dbf77767d8e9d468f6f7d355db1f35452c053a494795385420be942610abaa2",
+        "StakingPubKey": "a653f34413da465ec2fbee9b36f43baae5495580fc64a6b67bea2f766abcdec8ea53b33d16dd1f2f0779874c8c744e5d193a694c4336b5f67dd3190e660f278f3a9299ca2810632135b5dd5b6bae1f51735baa33c9dc3a89b00dc197224af1a2"
+    },
+    {
+        "Role": "access",
```
Comment (vishalchangrani) [inline nodeoperators/nodelist.json:3675]: unitedtraders

**Review by Kay-Zee — APPROVED** [review]

**Review by chandanworkacct — APPROVED** [review]

**Pull request merged**